cfgov/v1/wagtail_hooks.py

```diff
@@ -126,3 +127,9 @@ def expand_db_attributes(attrs, for_editor):
 @hooks.register('register_rich_text_link_handler')
 def register_cfgov_link_handler():
     return ('page', CFGovLinkHandler)
+
+
+@hooks.register('cfgov_context_handlers')
+def register_js_handler(*args):
```

Comment:
It might be more clear and helpful to others looking to make handlers like this if you specified the args explicitly, e.g.

```py
def register_js_handler(page, request, context):
```

Otherwise you have to go look at the `JSHandler` class to figure out what this is doing, and even then you have to go look at the base class.